data/resume/resume.json

```diff
@@ -5,48 +5,15 @@
     "label": "Frontend Software Engineer",
     "email": "jerry.batorski@gmail.com",
     "url": "https://jarekb84.github.io/me/",
-    "summaryOfQualifications": [
-      "11+ YOE as a software engineer, with a focus on the frontend",
-      "9+ YOE breaking down projects, communicating with stakeholders, delivering business value, and adapting to change",
-      "3+ YOE leading a frontend focused squad of software engineers"
-    ]
+    "summary": "Frontend focused software engineer, with over 11 years of experience, covering everything from implementing ideas to leading projects."
```

Comment:
I'll expand on this content later, in theory, I could dump in some sort of delimiter character here (maybe a `/n` or something` to denote breaking this back out into multiple lines. For now just summarized the prior bullet points.